src/tags/object/Ranker.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * libraries
+ */
+import React from 'react';
+import { inject, observer } from 'mobx-react';
+import { flow, types } from 'mobx-state-tree';
+
+import Registry from '../../core/Registry';
+import { AnnotationMixin } from '../../mixins/AnnotationMixin';
+import ProcessAttrsMixin from '../../mixins/ProcessAttrs';
+import Base from './Base';
+import { parseValue } from '../../utils/data';
+
+/**
+ * components
+ */
+import Ranker from '../../components/Ranker/Ranker';
+import { transformData } from '../../components/Ranker/createData';
+
+/**
+ * @name Ranker
+ * @meta_title Ranker Tag displays items that can be dragged and dropped between columns
+ * @meta_description Customize Label Studio by displaying and sorting results for machine learning and data science projects.
+ * @param {string} value Data field value containing JSON type for Ranker
```

Comment:
Docs should reflect attributes of this tag: `value`, `mode`, `title`. For `value` it can be something like "Data field containing a JSON with array of objects (id, title, body) to rank"

Reply:
```suggestion
 * @param {string} value    Data field containing a JSON with array of objects (id, title, body) to rank
```